types/swagger-express-middleware/index.d.ts

```diff
@@ -361,6 +363,11 @@ declare module "swagger-express-middleware" {
             externalDocs?: ExternalDocumentationObject
         }
 
+        export interface SwaggerParser {
```

Comment:
Instead of defining your own interface for this, could you just use `@types/swagger-parser`?

Reply:
`SwaggerParser` is a namespace as defined in `@types/swagger-parser`, and so it doesn't work in the callback prototype `(err: any, middleware: SwaggerMiddleware, api: SwaggerObject, parser: SwaggerParser): void;` where a type is required.

Reply:
ah, I see. In that case, you can use:
```ts
import * as parser from "swagger-parser";
type SwaggerParser = typeof parser.SwaggerParser;
```

Reply:
Thanks! I was not aware of this trick of converting namespaces into types.